packages/lodestar/src/sync/regular/naive/naive.ts

```diff
@@ -58,16 +58,18 @@ export class NaiveRegularSync implements IRegularSync {
     this.chain.on("processedBlock", this.onProcessedBlock);
     this.currentTarget = this.chain.forkChoice.headBlockSlot();
```

Comment:
can we store it in const headSlot, so code below is more clearer.

I was so confused that target larger than current slot means we are synced :smile: